components/modalize/view.coffee

```diff
@@ -65,6 +68,7 @@ module.exports = class Modalize extends Backbone.View
     @close(null, e) if e.which is 27
 
   maybeClose: (e) ->
+    return unless @backdropCloses
     @close(null, e) if $(e.target).hasClass('js-modalize-backdrop')
```

Comment:
I think this would read better by removing the negation and moving it into the existing conditional:

```coffeescript
maybeClose: (e) ->
  @close(null, e) if @backdropCloses and $(e.target).hasClass('js-modalize-backdrop')
```